src/azul/indexer/document.py

```diff
@@ -274,13 +292,30 @@ def to_index(self, value: T) -> T:
     def from_index(self, value: T) -> T:
         return value
 
+    @property
+    def json_type(self):
+        if self.n == JSON:
```

Comment:
This conditional deserves an explanation.

Reply:
I removed the conditional, the api_type for json should be an object, not a string.